naslib/search_spaces/nasbench201/conversions.py

```diff
@@ -107,25 +112,46 @@ def convert_naslib_to_str(naslib_object):
     Converts naslib object to string representation.
     """
 
-    ops_to_nb201 = {
-        "AvgPool1x1": "avg_pool_3x3",
-        "ReLUConvBN1x1": "nor_conv_1x1",
-        "ReLUConvBN3x3": "nor_conv_3x3",
-        "Identity": "skip_connect",
-        "Zero": "none",
+    cell = naslib_object.edges[2, 3].op
+    edge_op_dict = {
+        (i, j): OPS_TO_NB201[cell.edges[i, j]["op"].get_op_name] for i, j in cell.edges
     }
+    op_edge_list = [
+        "{}~{}".format(edge_op_dict[(i, j)], i - 1)
+        for i, j in sorted(edge_op_dict, key=lambda x: x[1])
+    ]
 
-    cell = naslib_object.edges[2, 3].op
+    return "|{}|+|{}|{}|+|{}|{}|{}|".format(*op_edge_list)
+
+def convert_str_to_op_indices(str_encoding):
+    """
+    Converts NB201 string representation to op_indices
+    """
+    nodes = str_encoding.split('+')
+    def get_op(x):
+        return x.split('~')[0]
+
+    node_ops = [list(map(get_op, n.strip()[1:-1].split('|'))) for n in nodes]
+
+    enc = []
+    for u, v in EDGE_LIST:
+        enc.append(OP_NAMES_NB201.index(node_ops[v-2][u-1]))
+
+    return tuple(enc)
+
+def convert_op_indices_to_str(op_indices):
```

Comment:
Accept this version of the functions with global already defined as it's more compact and consistent with other search spaces' code format (nb101 and tnb101)